CMake/sitkUseGTest.cmake

```diff
@@ -33,7 +33,9 @@ function(_sitk_gtest_use_gtest_source)
 
   set(BUILD_GTEST                 ON )
   set(BUILD_GMOCK                 OFF)
-
+  if ("${WIN32}")
+      set(gtest_disable_pthreads      ON CACHE INTERNAL "Disable use of pthreads in gtest")
+  endif()
```

Comment:
cmake on the windows machine I'm using is version 3.7.2. It is the version installed via msys2